doc/api/n-api.md

```diff
@@ -5479,8 +5479,8 @@ reference count reaches zero. In particular, `napi_call_threadsafe_function()`
 will return `napi_closing`, thus informing the threads that it is no longer
 possible to make asynchronous calls to the thread-safe function. This can be
 used as a criterion for terminating the thread. **Upon receiving a return value
-of `napi_closing` from `napi_call_threadsafe_function()` a thread must make no
-further use of the thread-safe function because it is no longer guaranteed to
```

Comment:
This reads better to me the way it was originally phrased.

Reply:
Yeah, that one can stand some improvement. I'll come up with something else or else change it back how it was.